samples/snapshot/pom.xml

```diff
@@ -30,8 +30,22 @@
       <artifactId>google-cloud-dialogflow-cx</artifactId>
       <version>0.10.2-SNAPSHOT</version>
     </dependency>
-    <!-- {x-version-update-end} -->
```

Comment:
This should be preserved.

Reply:
I've added this back in, but I'm going to keep this open until I'm ready to merge as OwlBot preprocessor sometimes changes things.